tests/test_mutations.py

```diff
@@ -397,3 +404,124 @@ def test_remove_all_milestone_issues(self):
             }
             }
         )
+
+
+class TestMutationRelatedObjectsWithOverrideSettings(BaseTestCase):
+    """Tests for creating and updating reverse side of FK and M2M relationships."""
+
+    @override_settings(GRAPHENE_DJANGO_PLUS={'MUTATIONS_INCLUDE_REVERSE_RELATIONS': False})
+    def test_create_milestone_issues_turned_off_related_setting(self):
+        """Test that a milestone can be created with a list of issues."""
+
+        milestone = 'release_1A'
+        self.assertIsNone(Milestone.objects.filter(name=milestone).exists())
+
+        project_id = to_global_id(ProjectType.__name__, self.project.id)
+        issue_id = to_global_id(IssueType.__name__, self.issues[0].id)
+
+        # Creating schema inside test run so that settings would be already modified
+        # since schema is generated on server start, not on execution
+        class MilestoneCreateMutation(ModelCreateMutation):
+            class Meta:
+                model = Milestone
+
+        class Mutation(graphene.ObjectType):
+            milestone_create = MilestoneCreateMutation.Field()
+
+        schema = graphene.Schema(
+            mutation=Mutation,
+        )
+        query = """
+            mutation milestoneCreate {
+              milestoneCreate (input: {
+                name: "%s",
+                project: "%s",
+                issues: ["%s"]
+              }) {
+                milestone {
+                  name
+                  issues {
+                    edges {
+                      node {
+                        name
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            """ % (milestone, project_id, issue_id)
+        result = schema.execute(query)
+        self.assertIsNone(Milestone.objects.filter(name=milestone).exists())
```

Comment:
`assertIsNone` seems strange here since you are called `.exists()`, which returns either `True` or `False.

Shouldn't it be `assertFalse`?